### ISSUE TITLE
Google sends invites/messages in the background to attendees of meetings originating in Outlook

### ISSUE THREAD

**Issue by dvirgassner** [phw198/OutlookGoogleCalendarSync#1659]
OGCS v2.9.0.0

To my horror, after syncing my Outlook calendar to Google (into a new calendar I opened under my regular account specifically for this purpose), Google began sending emails/invites to attendees of Outlook meetings -- even meetings where I'm NOT the organizer but rather an attendee!

An example of the header of such email (sensitive information hidden for privacy and replaced with bold text):

```
-----Original Appointment-----
From: <**My Google account address**> On Behalf Of <**My Outlook email address**>
Sent: Wednesday, 31 May 2023 21:39
To: <**long list of recipients**>
Cc: <**long list of recipients**>
Subject: Invitation from an unknown sender: FW: <**hidden for privacy**> @ Wed Oct 25, 2023 5pm - 8pm (IDT) (**email address of meeting organizer - not me!**)
When: Wednesday, 25 October 2023 17:00-20:00 Asia/Jerusalem.
Where: **hidden for privacy**
```

Is this a bug or perhaps I should have done something in advance in Google or in OGCS  to prevent this behavior?

**Comment by phw198** [comment]
Yep agreed, it's pretty horrific. But Google shows no sign of ever fixing this.

Unfortunately, it would seem you turned off [attendee email cloaking](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/FAQs#why-are-my-meeting-attendees-getting-notified-of-updates-to-events-in-google), which is on by default to protect again this spamming.

**Comment by AndrewMac1971** [comment]
In emails where I am a guest on outlook, i receive invite acceptances and rejections from other guests on outlook into my gmail.

I have the cloaking on and this still happens.

**Comment by phw198** [comment]
@AndrewMac1971 Should be easy enough to check in the Google item synced by OGCS whether the attendees have `.ogcs` appended to their email address. If not, run a full sync from O->G by pressing `Shift+Sync`, in case cloaking was turned on after the Google item(s) were synced.
